docs/installing-with-package-manager.md

```diff
@@ -45,6 +45,35 @@ emerge app-editors/lapce
 brew install lapce
 ```
 
+### nixpkgs
+
+You can find the packages [here](https://search.nixos.org/packages?channel=unstable&show=lapce&from=0&size=50&sort=relevance&type=packages&query=lapce):
+
+```bash
+# try with nix-shell
+nix-shell -p lapce
+
+# on NixOS
+nix-env -iA nixos.lapce
+
+# on non-NixOS installs, including macOS
+nix-env -iA nixpkgs.lapce
+
+# only if nix.settings.experimental-features is set to both "nix-command" and "flakes"
+# WARNING: THIS BREAKS nix-env, PROCEED AT YOUR OWN RISK. THIS ALSO INSTALLS FROM UNSTABLE BRANCH.
+nix profile install nixpkgs#hello
+```
+
+If you want to globally install Lapce as part of your NixOS system configuration (usually in `/etc/nixos/configuration`,
+append your `environment.systemPackages`:
+
+```diff
+   environment.systemPackages = [
+     # any other packages here
++    pkgs.lapce
+   ];
+```
```

Comment:
do we need this? nixos users should already know how to install packages

Reply:
Maybe not, I guess it is included in the NixOS Search result. (Please squash and merge when I pushed the edit via `github.dev`.)